specification/botservice/resource-manager/Microsoft.BotService/preview/2021-05-01-preview/botservice.json

```diff
@@ -1800,11 +1824,13 @@
       "properties": {
         "extensionKey1": {
           "type": "string",
-          "description": "The extensionKey1"
+          "description": "The extensionKey1",
+          "readOnly": true
```

Comment:
Have these breaking changes been signed off in both the older api versions ?

Reply:
Yes